exercises/concept/translation-service/.docs/instructions.md

```diff
@@ -99,11 +98,11 @@ service.request("jIyajbe'");
 
 ## 4. Fetch a translation, inspect the quality, or request it
 
-Implement the function `premium(text, quality)` for premium users, which fetches a translation, requests it if it's not available, and only returns it if it meets a certain threshold.
+Implement a premium user method `premium(text, quality)` to `api.fetch` a translation. If a translation is `notAvailable`, `api.request` the translation and `api.fetch` it after its been added to the _datastore_. The method should only return the translation if it meets a certain `quality` threshold.
```

Comment:
```suggestion
Implement a premium user method `premium(text, quality)` to fetch a translation.
If a translation is `NotAvailable`, request the translation and fetch it after its been added to the _datastore_.
The method should only return the translation if it meets a certain `quality` threshold.
```

Again, I'm not a fan of making the instructions so context-less that you no longer need the story.

Reply:
I think its easier for to learn promises, a tricky subject, if your explicit about the example API [if a side goal is not learning how to deal with sparsely documented API]. 

I think the changes you made give enough clarity that it won't give people API trouble.